.github/workflows/release_parallel.yml

```diff
@@ -0,0 +1,218 @@
+                  AZLE_VERBOSE=true npx azle install-global-dependencies --rust --wasi2ic
+
+            # TODO we should use some Action-specific bot account
+            - name: Configure git for publishing release
+              run: |
+                  git config --global user.name 'Jordan Last'
+                  git config --global user.email 'jordan.michael.last@gmail.com'
+                  git config --global commit.gpgsign true
+                  echo -n "$GPG_SIGNING_KEY" | base64 --decode | gpg --import
+                  git config --global user.signingkey C8B77BCBE16CD2B94B43F9C8757397B82D4ED7B0
+
+            - name: Update version and build templates
+              run: |
+                  VERSION=${{ steps.get-version.outputs.release-version }}
+                  sed -E -i "s/(\"version\": \")(.*)(\")/\1$VERSION\3/" package.json
+                  sed -E -i "s/(\"version\": \")(.*)(\")/\1$VERSION\3/" dfx_extension/extension.json
+                  npm install
+                  npx azle template
+                  npx azle template --experimental
+
+            - name: Publish to npm
+              run: |
+                  if [[ "${{ steps.get-version.outputs.release-version }}" == *"-rc."* ]]; then
+                    npm publish --tag next
+                  else
+                    npm publish
+                  fi
+
+            - id: get-test-infos
+              uses: ./.github/actions/get_test_infos
+              with:
+                  directories: |
+                      ./examples
+                      ./tests
+
+    update-dependencies:
+        needs: prepare-release
+        name: Update ${{ matrix.test.name }}
+        runs-on: ubuntu-latest
+        env:
+            GPG_SIGNING_KEY: ${{ secrets.GPG_SIGNING_KEY }}
+            GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        strategy:
+            fail-fast: false
+            matrix:
+                test: ${{ fromJson(needs.prepare-release.outputs.test-infos) }}
+        steps:
+            - uses: actions/checkout@v4
+              with:
+                  ref: ${{ github.event.pull_request.head.ref || github.ref }}
+                  token: ${{ secrets.LASTMJS_GITHUB_TOKEN || github.token }}
+
+            - id: get-node-version
+              uses: ./.github/actions/get_node_version
+
+            - uses: actions/setup-node@v4
+              with:
+                  node-version: ${{ steps.get-node-version.outputs.node-version }}
+
+            - id: get-dfx-version
+              uses: ./.github/actions/get_dfx_version
+
+            - name: Install dfx
+              run: |
+                  # Install dfx (Note: dfx must be installed before `npx azle` because the azle installation process requires dfx)
+                  src/build/stable/commands/install_global_dependencies/install_dfx.sh ${{ steps.get-dfx-version.outputs.dfx-version }}
+                  echo "$HOME/.local/share/dfx/bin" >> $GITHUB_PATH
+
+            - name: Update dependencies
```

Comment:
Can we rename this job to explain what we're updating? To me it was unclear on if this was updating some kind of azle dependencies through installation or something...so it's updating the configurationish files of the examples/tests

Reply:
sure

Reply:
done